lib/shopify_transporter/shopify/order.rb

```diff
@@ -65,7 +67,7 @@ def to_csv
       LINE_ITEM_PREFIX = 'lineitem_'
 
       LINE_ITEM_ATTRIBUTES = %w(
-        name quantity price sku requires_shipping taxable fulfillment_status
+        name quantity price discount compare_at_price sku requires_shipping taxable fulfillment_status
```

Comment:
The transporter sample csv has compare_at_price, so I think this is fine.

If we decide to leave it, we should update the spec to have this column as well.

Can you double check that the tests are correct for this? The headers are different from the test and they're still passing - I think it's because they have the same number of columns.